__tests__/app.test.js

```diff
@@ -1,8 +1,47 @@
 const request = require('supertest');
 const db = require('../db/connection');
 const seed = require('../db/seeds/seed');
-const testTopicData = require('../db/data/test-data/topics');
+const {articleData, commentData, topicData, userData} = require('../db/data/test-data/index');
 const { app } = require("../app");
 
 afterAll(() => db.end());
-beforeEach( () => seed(testTopicData));
\ No newline at end of file
+beforeEach( () => seed({topicData, userData, articleData, commentData}));
+
+describe('1. GET api topics', () => {
+    describe('a. status 200 & Data', () => {
+        it('sends a 200 request back to the user', () => {
+            return request(app)
+                .get('/api/topic')
+                .expect(200)
+        })
+        it('returns an array back to the user', () => {

```

Comment:
it's okay if you prefer to write the tests this way but just so you're aware it is also fine if you want to combine these tests, the `.then` can simply follow straight on from the `.expect(200)`